master/buildbot/test/unit/www/test_hooks_github.py

```diff
@@ -564,6 +564,9 @@
 
   ]
 }"""
+
+gitJsonPayloadNotFound = b"""{"message":"Not Found"}"""
```

Comment:
I would inline such small test data. Duplication is even beneficial in tests as it makes tests self-contained and thus easier to understand and change.

Reply:
Cool, I wasn't sure which you would prefer so I went with making it shared. Since you merged I'll just keep that in mind for my next PR (I'm assuming I'll have one at some point).